drivers/flash/soc_flash_nrf.c

```diff
@@ -80,6 +80,18 @@ static struct k_sem sem_lock;
 #define SYNC_UNLOCK()
 #endif
 
+#ifdef CONFIG_NRF52_ANOMALY_242_WORKAROUND
+#include <hal/nrf_power.h>
+static int suspend_pofwarn(void);
+static void restore_pofwarn(void);
+
+#define SUSPEND_COLIDING_SYSTEMS() suspend_pofwarn()
```

Comment:
That are strange names those macros have. It should be "colliding" probably, but that too is weird. Maybe "conflicting" or something.